reader.go

```diff
@@ -64,13 +70,24 @@ func NewReader(r io.Reader) *Reader {
 // on the first character of each line. It also enforces FED Wire formatting rules and returns
 // the appropriate error if issues are found.
 func (r *Reader) Read() (File, error) {
+
+	stripNewLines := func(input string) string {
+		// windows
+		input = strings.ReplaceAll(input, "\r\n", "")
+
+		// mac, linux
+		input = strings.ReplaceAll(input, "\n", "")
+
+		return input
+	}
```

Comment:
Do we need to this remove newlines from the middle of strings? I would think `strings.TrimSpace` can handle this.

Reply:
fixed